mmv1/products/essentialcontacts/api.yaml

```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: beta
     base_url: https://essentialcontacts.googleapis.com/v1beta1/
```

Comment:
if beta is deprecated should we delete beta endpoint? 

The beta provider will switch to use the ga enpoint.

Reply:
Yeah, fixed